[tsc] Cannot find name 'Deno'.

# Bug report

## Description / Observed Behavior

`src/use-swr.ts:30:13 - error TS2304: Cannot find name 'Deno'.`

## Expected Behavior
Error not found.

## Repro Steps / Code Example

Or share your code snippet or a [CodeSandbox](https://codesandbox.io) link is also appreciated!

## Additional Context
Fork master branch and `yarn run types:check`

--- a/src/use-swr.ts
+++ b/src/use-swr.ts
@@ -24,9 +24,9 @@ import {
   updaterInterface
 } from './types'
 
-// @ts-ignore
 const IS_SERVER =
   typeof window === 'undefined' ||
+  // @ts-ignore
   !!(typeof Deno !== 'undefined' && Deno && Deno.version && Deno.version.deno)
 
 // polyfill for requestAnimationFrame

--- a/src/use-swr.ts
+++ b/src/use-swr.ts
@@ -25,7 +25,9 @@ import {
 } from './types'
 
 // @ts-ignore
-const IS_SERVER = typeof window === 'undefined' || !!(typeof Deno !== 'undefined' && Deno && Deno.version && Deno.version.deno)
+const IS_SERVER =
+  typeof window === 'undefined' ||
+  !!(typeof Deno !== 'undefined' && Deno && Deno.version && Deno.version.deno)
 
 // polyfill for requestAnimationFrame
 const rAF = IS_SERVER
@@ -301,7 +303,7 @@ function useSWR<Data = any, Error = any>(
   useDebugValue(stateRef.current.data)
 
   const rerender = useState(null)[1]
-  let dispatch = useCallback(payload => {
+  let dispatch = useCallback((payload: actionType<Data, Error>) => {
     let shouldUpdateState = false
     for (let k in payload) {
       if (stateRef.current[k] === payload[k]) {

